src/chocolatey.tests.integration/context/dependenciesbeforemodify/hasdependencywithbeforemodify/1.0.0/hasdependencywithbeforemodify.nuspec

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>hasdependencywithbeforemodify</id>
+        <version>1.0.0</version>
+        <title>hasdependencywithbeforemodify</title>
+        <authors>__REPLACE_AUTHORS_OF_SOFTWARE__</authors>
+        <owners>__REPLACE_YOUR_NAME__</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>__REPLACE__</description>
+        <summary>__REPLACE__</summary>
```

Comment:
Should we add the contents of the `purpose.txt` document to the description, summary, or both? I realize a lot of our test packages don't have this, but would be helpful when looking at the packages in a repository to understand the purpose.

Reply:
We could do? TBH if we wanna do this I might suggest we just ditch purpose.txt files entirely and put it in the package summary / description instead.

If we wanna do that I'd suggest we do them all in one PR though, get 'em all done in one sweep.

Reply:
That makes sense. I'll mark this conversation resolved and add a note to #2828 since it feels like that'd be where we do it.